src/Engine/ProtoCore/Properties/Resources.en-US.resx

```diff
@@ -117,9 +117,81 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="attributeArgMustBeConstant" xml:space="preserve">
+    <value>An attribute argument must be a constant expression</value>
+  </data>
+  <data name="baseIsExpectedToCallBaseConstructor" xml:space="preserve">
+    <value>'base' is expected here to call base constructor.</value>
+  </data>
+  <data name="CheckIfTheElementsInListAreSameType" xml:space="preserve">
+    <value>Checks if all the elements in the specified list have the same type.</value>
+  </data>
+  <data name="ChecksIfLengthsAreSameInMultiDimentionalList" xml:space="preserve">
+    <value>Checks if each of rows in multidimentional list has the same number of elements</value>
```

Comment:
Checks if each of rows in multidimensional list has the same number of elements.